.github/workflows/task-deploy-worker.yml

```
@@ -39,34 +39,23 @@
-      - name: NPM Install
-        run: npm install --silent
+          cache: 'pnpm'
+      - name: Install dependencies
+        run: pnpm install --silent
 
       - name: nx Install
-        run: npm install -g nx --silent
+        run: pnpm add -g nx --silent
 
       - name: Get affected apps
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:86: Double quote to prevent globbing and word splitting

(shellcheck)